src/components/VCopyButton.vue

```diff
@@ -55,10 +55,21 @@ export default defineComponent({
       }, 2000)
 
       e.clearSelection()
+
+      /* Set the focus back on the button */
+      const button = document.getElementById(props.id)
+      if (button) {
+        button.focus()
+      }
     }
     const onCopyError = (e: Clipboard.Event) => {
       emit('copy-failed')
       e.clearSelection()
+      /* Set the focus back on the button */
+      const button = document.getElementById(props.id)
+      if (button) {
+        button.focus()
+      }
```

Comment:
The fix is working great, thank you, @ramadanomar !
Can we extract the lines 67-72 into a function, and reuse it for lines 57-63, too?